### DISCUSSION TITLE
OBS 28 Apple Silicon version being identified as an iOS app

### DISCUSSION THREAD

**Discussion by infinitepower18** [obsproject/obs-studio#6957] [General]
I have noticed this with other apps too but don't know whether it's the problem with the app or macOS itself... but the beta version of OBS 28 for Mac arm64 is being identified as an iOS app.

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/44692189/183020258-6fa5c1ac-1f18-4c5e-9332-0f1b8ca34c26.png">

**Comment by derrod** [comment]
I mean, does it matter? Maybe that's just what happens if an app is not universal but pure ARM?

**Comment by WizardCM** [comment]
This does seem to be the case for GammaRay and VLC too. I have a feeling it doesn't functionally matter, but I did notice my local Debug build is correctly identified as Apple Silicon.

Probably not something we'll attempt to fix unless it introduces major problems.

**Comment by WizardCM** [comment]
We have determined that this happens when built with anything that isn't Xcode, such as Ninja.

**Reply by PatTheMav** [reply]
To follow-up on this, it doesn't seem to have a negative effect, but we reached out to Apple asking for some pointers as to how/why it happens.

**Comment by PatTheMav** [comment]
Will be fixed when https://github.com/obsproject/obs-studio/pull/6997 is merged.
